framework/src/test/java/viewpointtest/quals/Lost.java

```diff
@@ -0,0 +1,22 @@
+package viewpointtest.quals;
+
+import org.checkerframework.framework.qual.SubtypeOf;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * {@link Lost} is type annotation that indicates the relationship can not be expressed. It is the
```

Comment:
```suggestion
 * The {@link Lost} qualifier indicates that a relationship cannot be expressed. It is the
```

Might need to re-flow.